src/Browscap/Data/Factory/UseragentFactory.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * This file is part of the browscap package.
```

Comment:
No need for copyright notices in every file (just creates stupid maintenance every year)